README.md

```diff
@@ -1,14 +1,42 @@
-![](https://github.com/karam084/online-course-website)
+![](https://github.com/karam084/online-courses-capstone)
 
 # Project Name
 
-Online Course WebSite
+Online Course capstone project  
 
-> One paragraph statement about the project.
+![Screenshot](https://user-images.githubusercontent.com/77942746/152469213-7df2dcad-1368-4063-ac6a-75680eac8e6d.png)
 
-![screenshot](/home/)
+![Screenshot](https://user-images.githubusercontent.com/77942746/152469254-f1c73ab5-8722-4f59-912c-b644cbd9a59d.png)
 
-Additional description about the project and its features.
+![Screenshot](https://user-images.githubusercontent.com/77942746/152469308-b34826ac-f714-4697-b1b9-6f3dee34e91b.png)
+
+
+![Screenshot](https://user-images.githubusercontent.com/77942746/152469343-4c3e0ca5-3d63-4de1-abd4-a06a506ee9e3.png)
+
+
+![Screenshot](https://user-images.githubusercontent.com/77942746/152469396-484bc679-ff65-4a45-b5e2-32d952f7ced4.png)
+
+![Screenshot](https://user-images.githubusercontent.com/77942746/152469449-dbd4ac9f-aa17-4907-9a94-8dae571d45a0.png)
+
+![Screenshot](https://user-images.githubusercontent.com/77942746/152469473-c6e313b6-1bdc-4fce-935c-84ea64f9b1df.png)
+
+![Screenshot](https://user-images.githubusercontent.com/77942746/152469505-6a7d7702-15a3-4f53-9fa7-5a448d38779c.png)
+
+![Screenshot](https://user-images.githubusercontent.com/77942746/152469546-282ff7b6-2b8b-4fdb-9824-06feee00f4a1.png)
+
+![Screenshot](https://user-images.githubusercontent.com/77942746/152469582-dea4a35e-3ae3-4370-9fb9-9918ea95795c.png)
+
+![Screenshot](https://user-images.githubusercontent.com/77942746/152469620-46a2672d-fbea-4f73-a3fd-7998eaaaf64c.png)
+
+![Screenshot](https://user-images.githubusercontent.com/77942746/152469662-9e96c0b8-5180-42d9-b473-0ebc61cd989a.png)
+
+
+
+
+
+### Video link about project
+
+(https://www.loom.com/share/32a0ee3361cd4f6dbad65f5b52496d5d)
 
 ## Built With
 
```

Comment:
- [ ] I can't highlight the acknowledgment section so I'm highlighting this in place of the acknowledgment section. It is required that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).